src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="https://www.google.com/search"
+          method="get"
+          class="container container__big"
+          data-qa="big"
+    >
```

Comment:
Need to fix code style everywhere
```suggestion
    <form
      action="https://www.google.com/search"
      method="get"
      class="container container__big"
      data-qa="big"
    >
```